### PR TITLE
Wiki Integration

### PR DESCRIPTION
**Pull request by EmeraldSnorlax**
Downloads a local copy of the wiki on bot start. Staff members can also update the bot's copy of the wiki without restarting by running i!wikiresync

now if someone were to actually finish the wiki LMAO

btw this needs cli git to be installed on your heroku instance

### CODE REVIEW

File: commands/Staff/wikiResync.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  constructor (...args) {
+    super(...args, {
+      name: 'wikiResync',
+      enabled: true,
+      runIn: ['text'],
+      cooldown: 30,
+      bucket: 1,
+      aliases: ['wikiupdate', 'wikipull', 'wikisync', 'updatewiki', 'syncwiki'],
+      permissionLevel: 0,
+      requiredPermissions: ['SEND_MESSAGES', 'CHANGE_NICKNAME'], // Lazy way to check for a staff role LMAO
```
Comment (LeafHacker) [inline commands/Staff/wikiResync.js:15]: Don't think all staff have this perm anymore... Might be better to check agaist a list of role IDs?

**Comment by LeafHacker** [comment]
I assume heroku instances have git installed by default? Not sure if/where that's documented.

My main concern would be how much of our quota cloning the wiki will use up, also need to ensure that is cached across instance restarts etc

**Comment by EmeraldSnorlax** [comment]
i have no idea if git comes on heroku instances, but it shouldnt use that much bandwidth as my local copy is sitting at 214kb

should i fix the permission checking, or just close the pr?

**Comment by LeafHacker** [comment]
Closing due to the go rewrite

**Pull request closed without merging**